src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -997,10 +1124,12 @@ await DownloadFileAsync(file.FilePath, destination, cancellationToken)
             bool? canRestrictMembers = default,
             bool? canPinMessages = default,
             bool? canPromoteMembers = default,
-            CancellationToken cancellationToken = default
+            CancellationToken cancellationToken = default,
+            bool? isAnonymous = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.